### PR TITLE
Hotfix/change role titles

### PR DESCRIPTION
**Pull request by stuartjennings-up**

### CODE REVIEW

File: src/core/decorators.py [hunk 1]
```diff
@@ -86,7 +86,7 @@ def wrap(request, *args, **kwargs):
 		if 'press-editor' in user_roles or 'series-editor' in user_roles or 'book-editor' in user_roles or 'production-editor' in user_roles:
 			return function(request, *args, **kwargs)
 		else:
-			messages.add_message(request, messages.ERROR, 'You need to have Press Manager, Book Manager or Series Editor level permission to view this page.')
+			messages.add_message(request, messages.ERROR, 'You need to have Press Editor, Press Editor or Series Editor level permission to view this page.')
```
Comment (craiga) [inline src/core/decorators.py]: "Press Editor, Press Editor or Series Editor"? Should these strings just be "Press Editor or Series Editor" now?

Comment (craiga) [reply src/core/decorators.py]: Or are they "book editors" instead of "book managers" now?

File: src/templates/editor/submission_details.html [hunk 2]
```diff
@@ -1,11 +1,11 @@
-<h3>EDITORS 
+<h3>BOOK EDITOR
```
Comment (craiga) [inline src/templates/editor/submission_details.html]: Should this be plural? And where's the closing `h3` tag?

**Review by craiga — CHANGES_REQUESTED** [review]

File: src/templates/editor/submission_details.html (hunk 2 above)
Comment (stuartjennings-up) [reply src/templates/editor/submission_details.html]: Ah well spotted with the plural. The closing tag is further down as the title includes a link. Will make this more obvious in formatting

File: src/core/decorators.py (hunk 1 above)
Comment (stuartjennings-up) [reply src/core/decorators.py]: Yes should be Book Editor, thanks

**Review by craiga — APPROVED** [review]

**Pull request merged**